src/seedu/addressbook/commands/HelpCommand.java

```diff
@@ -15,14 +15,15 @@
     public CommandResult execute() {
         return new CommandResult(
                 AddCommand.MESSAGE_USAGE
-                + "\n" + DeleteCommand.MESSAGE_USAGE
-                + "\n" + ClearCommand.MESSAGE_USAGE
-                + "\n" + FindCommand.MESSAGE_USAGE
-                + "\n" + ListCommand.MESSAGE_USAGE
-                + "\n" + ViewCommand.MESSAGE_USAGE
-                + "\n" + ViewAllCommand.MESSAGE_USAGE
-                + "\n" + HelpCommand.MESSAGE_USAGE
-                + "\n" + ExitCommand.MESSAGE_USAGE
+                        + "\n" + DeleteCommand.MESSAGE_USAGE
```

Comment:
Are there tabs here (and in the following Case statements?)
Please see https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#layout